apps/print/frontend/src/components/toolbar.tsx

```diff
@@ -147,29 +159,61 @@ function PrinterStatus({ status }: { status: PrinterStatusType }) {
   return <PrinterConnectionStatus connected={connected} />;
 }
 
-function UsbStatus({ status }: { status: UsbDriveStatus }) {
+function BatteryStatus({ status }: { status?: BatteryInfo }) {
+  return (
+    <Row style={{ gap: '0.25rem', alignItems: 'center' }}>
+      {getBatteryIcon(status, true)}
+      {status && !status.discharging && (
+        <Icons.Bolt style={{ fontSize: '0.8em' }} color="inverse" />
+      )}
+      {status && format.percent(status.level)}
+      {status && status.level < 0.25 && status.discharging && (
+        <Icons.Warning color="inverseWarning" />
+      )}
+    </Row>
+  );
+}
```

Comment:
No need for this VxPrint push but noting - we should probably extract this out for the sake of VxPrint and VxPollBook eventually.